src/RefractAPI.cc

```diff
@@ -177,15 +213,18 @@ namespace drafter {
     }
 
     template<typename T>
-    refract::IElement* ExtractParameter(const snowcrash::Parameter& parameter)
+    refract::IElement* ExtractParameter(const SectionInfo<snowcrash::Parameter>& parameter)
     {
         refract::IElement* element = NULL;
 
-        if (parameter.values.empty()) {
-            element = refract::IElement::Create(LiteralTo<T>(parameter.exampleValue));
+        if (parameter.section.values.empty()) {
+            element = refract::IElement::Create(LiteralTo<T>(parameter.section.exampleValue));
+            AttachSourceMap(element, MAKE_SECTION_INFO(parameter, exampleValue));
 
-            if (!parameter.defaultValue.empty()) {
-                element->attributes[SerializeKey::Default] = refract::IElement::Create(LiteralTo<T>(parameter.defaultValue));
+            if (!parameter.section.defaultValue.empty()) {
+                refract::IElement* defaultElement = refract::IElement::Create(LiteralTo<T>(parameter.section.defaultValue));
+                AttachSourceMap(defaultElement, MAKE_SECTION_INFO(parameter, defaultValue));
+                element->attributes[SerializeKey::Default] = defaultElement;
```

Comment:
Could have used `LiteralToRefract` in just one line instead of the above 3 lines.